airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQuerySQLNameTransformer.java

```diff
@@ -41,6 +41,7 @@ public String getNamespace(final String input) {
     return normalizedName;
   }
 
+  @Deprecated
   public String getTmpTableName(final String streamName, final String randomSuffix) {
```

Comment:
guessing it's a lot of work to properly delete this since it's declared in the superclass? (... it needs an `@Override` annotation, I had to check the source to find that)

maybe worth making it just `throw new UnsupportedOperationException`? unless we think we'll want to start using tmp tables for something in the future

Reply:
TBH I punted on this one when it started reaching into the CDK - `UnsupportedOperationException` seems like the right call